src/Ocis.php

```diff
@@ -480,6 +481,40 @@ public function createDrive(
         );
     }
 
+    /**
+     * Undocumented function
+     *
+     * @param string $search
+     * @param OrderDirection $orderBy
+     * @param boolean $expandMembers
+     * @return array<int,Group>
+     */
+    public function listGroups(string $search = "",OrderDirection $orderBy = OrderDirection::ASC,bool $expandMembers = false)
+    {
+        $apiInstance = new GroupsApi($this->guzzle, $this->graphApiConfig);
+        if($orderBy !== []) {
+            $orderBy = $orderBy->value === "asc" ? "displayName" : "displayName desc";
```

Comment:
I would suggest not to rewrite the variable, because that changes the type.

And can we get rid of the hard-coded string e.g.?
`$orderString = $orderBy->value === OrderDirection::ASC->value ? "displayName" : "displayName desc";`